examples/api/lib/widgets/basic/indexed_stack.0.dart

```diff
@@ -0,0 +1,152 @@
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({super.key});
+
+  static const String _title = 'Flutter Code Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: _title,
+      home: Scaffold(
+        appBar: AppBar(title: const Text(_title)),
+        body: const MyStatefulWidget(),
+      ),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  const MyStatefulWidget({super.key});
+
+  @override
+  State<MyStatefulWidget> createState() => _MyStatefulWidgetState();
+}
+
+class _MyStatefulWidgetState extends State<MyStatefulWidget> {
+  List<String> names = <String>['Dash', 'John', 'Mary'];
+  int index = 0;
+  final TextEditingController fieldText = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: <Widget>[
+        SizedBox(
+          width: 300,
+          child: TextField(
+            decoration: const InputDecoration(
+              border: OutlineInputBorder(),
+              hintText: 'Enter the name for a person to track',
+            ),
+            onSubmitted: (String value) {
+              setState(() {
+                names.add(value);
+              });
+              fieldText.clear();
+            },
+            controller: fieldText,
+          ),
+        ),
+        const SizedBox(height: 50),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            GestureDetector(
+              onTap: () => setState(() {
+                if (index == 0) {
+                  index = names.length - 1;
+                } else {
+                  index--;
+                }
+              }),
+              child: const Icon(key: Key('gesture1'), Icons.chevron_left),
+            ),
+          ],
+        ),
+        const SizedBox(height: 50),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            GestureDetector(
+              onTap: () => setState(() {
+                if (index == 0) {
+                  index = names.length - 1;
+                } else {
+                  index--;
+                }
+              }),
+              child: const Icon(key: Key('gesture1'), Icons.chevron_left),
+            ),
+            Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                IndexedStack(
+                  index: index,
+                  children: <Widget>[for (String name in names) PersonTracker(name: name)],
+                )
+              ],
+            ),
+            GestureDetector(
+              onTap: () => setState(() {
+                if (index == names.length - 1) {
+                  index = 0;
+                } else {
+                  index++;
```

Comment:
[Prefer `+=` over `++`](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-over)